schemas/v3.0/schema.yaml

```diff
@@ -1,6 +1,6 @@
-id: https://spec.openapis.org/oas/3.0/schema/2021-09-28
+id: https://spec.openapis.org/oas/3.0/schema/2024-10-10
 $schema: http://json-schema.org/draft-04/schema#
-description: The description of OpenAPI v3.0.x documents, as defined by https://spec.openapis.org/oas/v3.0.3
```

Comment:
@karenetheridge I realize you are not the person who put the "as defined by ...v3.0.N" here, but I'd really like to drop it. This is for 3.0. All of 3.0.